hr_af_holidays/data/resource.calendar.leaves.csv

```diff
@@ -0,0 +1,29 @@
+id,name,date_to,date_from,calendar_id/id,resource_id/id
+__import__.af_calendar_leaves_2020-4-9,Skärtorsdag,2020-04-10 00:00:00,2020-04-09 14:00:00,resource.resource_calendar_std,
```

Comment:
Consider changing "__import__" to the technical name of the module "hr_af_holidays" in the external id.